node-fs-writefile/note-to-self.js

```diff
@@ -0,0 +1,11 @@
+import { writeFile } from 'node:fs/promises';
+
+const [, , ...args] = process.argv;
```

Comment:
This will make `args` be an array. On line 6 you treat it as if it were a string. This basically works, but it doesn't quite write exactly what you want.